bin/ch/stdafx.h

```diff
@@ -239,7 +239,8 @@ class AutoString
         {
             return data_wide;
         }
-        NarrowStringToWideDynamic(data, &data_wide);
+        size_t destCount;
+        utf8::NarrowStringToWide<utf8::malloc_allocator>(data, length, &data_wide, &destCount);
```

Comment:
This probably leaked before too, but can we add a `~AutoString` that just calls free(data_wide), with the note that we are using free explicitly because its allocated with malloc here?

Reply:
Actually there's already a `free(data_wide)` in the `~AutoString` at L274: https://github.com/xiaoyinl/ChakraCore/blob/e2a74acc5f857303610b911a6ef14c99ae81c1b5/bin/ch/stdafx.h#L274

Reply:
Oops, I only looked above, not below. Sorry!